.github/ISSUE_TEMPLATE/new_package.yml

```diff
@@ -69,34 +69,35 @@ body:
       label: Category
       description: Which category should this tool be installed to?
       options:
+        - Active Directory
         - Android
         - Cloud
+        - Command & Control
         - Debuggers
         - Delphi
         - Disassemblers
-        - dotNet
+        - Evasion
+        - Exploitation
         - Forensic
         - Hex Editors
+        - Information Gathering
+        - InnoSetup
         - Java
         - Javascript
         - Networking
         - Office
         - PDF
         - PE
+        - Password Attacks
         - PowerShell
         - Python
         - Text Editors
         - Utilities
         - VB
-        - Active Directory
-        - Command & Control
-        - Evasion
-        - Exploitation
-        - Information Gathering
-        - Password Attacks
         - Vulnerability Analysis
         - Web Application
         - Wordlists
+        - dotNet
```

Comment:
if A-Z why is dotNet at the end?

Reply:
Because it is not capitalized as the rest. I used `:sort` in vim. Should I put in in the D (ignoring capitalization)?

Reply:
I have updated it using `:sort i` (case insensitive)